.github/workflows/infra.yml

```diff
@@ -19,9 +19,9 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: 12
-    - uses: microsoft/playwright-github-action@v1
```

Comment:
Not related to this PR: We use `microsoft/playwright-github-action` in quite a few other places in this repository. Can we fully remove it from the repo?

Reply:
yess good point